### ISSUE TITLE
Cannot connect to AP with Kali & Raspberry Pi 4

### ISSUE THREAD

**Issue by Joerny** [wifiphisher/wifiphisher#1271]
**Version**: commit 46bdf134d838bb33e1272c598a67c8cf085c5d8d (latest version from GitHub)
Raspberry Pi 4
Kali Linux 2020.1-rpi3-nexmon

**Description**: Starting the program with `wifiphisher -iNM`. A network can be chosen and the AP can be seen. But the client cannot connect to the AP (tested with Mac and iPhone). No error message seems to be displayed.

**Script output**: 

```
root@kali:~# wifiphisher -iNM --logging
[*] Starting Wifiphisher 1.4GIT ( https://wifiphisher.org ) at 2020-02-18 13:37
[+] Timezone detected. Setting channel range to 1-13
[+] Selecting wfphshr-wlan0 interface for the deauthentication attack
[+] Selecting wlan0 interface for creating the rogue Access Point
[+] Sending SIGKILL to wpa_supplicant
[+] Sending SIGKILL to dhclient
[+] Sending SIGKILL to NetworkManager
[*] Cleared leases, started DHCP, set up iptables
[+] Selecting OAuth Login Page template
[*] Starting the fake access point...
[*] Starting HTTP/HTTPS server at ports 8080, 443
[+] Show your support!
[+] Follow us: https://twitter.com/wifiphisher
[+] Like us: https://www.facebook.com/Wifiphisher
[+] Captured credentials:
```

```
root@kali:~# more wifiphisher.log
2020-02-18 13:37:28,042 -        wifiphisher.pywifiphisher - INFO - Starting Wifiphisher
2020-02-18 13:37:28,619 -        wifiphisher.common.opmode - INFO - Starting OP_MODE5 (0x5)
2020-02-18 13:37:28,863 -        wifiphisher.pywifiphisher - INFO - Selecting wfphshr-wlan0 for deauthentication and wlan0 for the rogue Access Point
2020-02-18 13:37:28,865 -        wifiphisher.pywifiphisher - INFO - Unblocking interfaces
2020-02-18 13:37:28,970 -        wifiphisher.pywifiphisher - INFO - Killing all interfering processes
2020-02-18 13:37:45,742 -        wifiphisher.pywifiphisher - INFO - Selecting OAuth Login Page template
```

**Comment by sophron** [comment]
Hi @Joerny,

What exactly happens when you are selecting the network? Can you verify that dnsmasq is running? `ps -aux | grep dnsmasq`?

**Comment by Joerny** [comment]
On Mac and iPhone there is a message that it cannot connect to that network.

```
root@kali:~# ps aux | grep dnsmasq
root       939  0.0  0.0      0     0 pts/0    Z+   16:20   0:00 [dnsmasq] <defunct>
nobody     943  0.0  0.0   8372   280 ?        S    16:20   0:00 dnsmasq -C /tmp/dnsmasq.conf
root       974  0.0  0.0   5080   536 pts/1    S+   16:22   0:00 grep dnsmasq
```

**Comment by bl4ckgh0st** [comment]
Hi, I have almost the same issue. Kali running on a raspberry pi 4. I don't know if I need to open a new issue.
Connection to the rogue AP is difficult from an iPhone or a Macbook. It is very long, and when the device is finally connected, the page asking for credentials doesn't appear. (Well, to be honest, it appeared once or twice over 20 tests.)
I tried a lot of configurations, with or without a password, faking an existing AP or creating a new one, and the results are the same.

However, connecting to the rogue AP from an Android device works fine.

**Comment by phspade** [comment]
same here

`Linux kali 4.19.93-Re4son-v8l+ #1 SMP PREEMPT Tue Jan 7 22:53:24 UTC 2020 aarch64 GNU/Linux`

![2020-04-30_17-29-19](https://user-images.githubusercontent.com/18681023/80695754-ef5bee00-8b08-11ea-8997-ca4e88b708eb.png)

Latest Wifiphisher installed using manual install (git clone blah blah blah)
